modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/model-source.mustache

```diff
@@ -245,26 +245,29 @@ void {{classname}}::fromJson(web::json::value& val)
         {{#isString}}
         {{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));
         {{/isString}}
+        {{#isByteArray}}{{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));{{/isByteArray}}
         {{^isString}}
         {{#isDateTime}}
         {{setter}}(ModelBase::dateFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));
         {{/isDateTime}}
-        {{^isDateTime}}
+        {{^isDateTime}}{{^isByteArray}}
         if(!val[utility::conversions::to_string_t("{{baseName}}")].is_null())
         {
             {{{dataType}}} newItem({{{defaultValue}}});
             newItem->fromJson(val[utility::conversions::to_string_t("{{baseName}}")]);
             {{setter}}( newItem );
         }
-        {{/isDateTime}}
+        {{/isByteArray}}{{/isDateTime}}
         {{/isString}}
     }
     {{/required}}
     {{#required}}
     {{#isString}}
     {{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));
-    {{/isString}}
-    {{^isString}}
+    {{/isString}}{{#isByteArray}}
+    {{setter}}(ModelBase::stringFromJson(val[utility::conversions::to_string_t("{{baseName}}")]));
```

Comment:
same

Reply:
Will check.

Reply:
See above. Also tested using spec from #407

Reply:
I'm surprised with this suspicious use of the undefined ModelBase::stringFromJson because I generated a cpprestsdk client with a heavy YAML input file and the build + partial link (lib generation) succeeded. May be I missed something ?

Reply:
@CyrilleBenard 
:) 
`ModelBase::stringFromJson` is there 
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/modelbase-header.mustache#L52

Reply:
@etherealjoy Yes sure but so, what's the @stkrwork remark about ? 
> stringFromJson does not exist, according to your header

I'm a little lost :
* I checked your current PR and I found no compilation issue
* @stkrwork said : stringFromJson does not exist

:fearful:

Reply:
i didnt see that that file was already part of the cpp rest sdk. i thought he was purely working on pistache in the pr, so that is why i was confused, because i checked the pistache file for modelbase-header